src/avg.js

```diff
@@ -10,13 +10,21 @@ let sum = require('./sum');
  * @returns {number} The average
  */
 function avg(array) {
+  let count = 0
+  for (let element in array) {
```

Comment:
Never use `in`. Never ever ever. You've never seen me use it, only warn you away from using it. There's no example anywhere in any of the material I've shared that uses `for ... in`.

Don't use it!

Reply:
See:

- https://github.com/jfarmer/intro-javascript/blob/master/Iteration/Example-Loops.md
- https://github.com/jfarmer/intro-javascript/blob/master/Iteration/For-Of.md